app/templates/macros/nav_macros.html

```diff
@@ -122,3 +122,57 @@ <h3 class="ui header">Bulk</h3>
         {{ desktop_nav(current_user, endpoints=endpoints, count=count) }}
     </nav>
 {% endmacro %}
+) }}" class="item">Log out</a>
```

Comment:
did you make additional changes to this file?
i think maybe something went wrong on a merge?